backend/shared/src/supabase/answers.ts

```diff
@@ -1,6 +1,16 @@
-import { SupabaseDirectClient } from 'shared/supabase/init'
+import { SupabaseDirectClient, pgp } from 'shared/supabase/init'
 import { convertAnswer } from 'common/supabase/contracts'
 import { groupBy } from 'lodash'
+import { Answer } from 'common/answer'
+import { bulkInsert, updateData, insert as upsert } from './utils'
+import { randomString } from 'common/util/random'
+import { removeUndefinedProps } from 'common/util/object'
+import { millisToTs } from 'common/supabase/utils'
+
+export const getAnswer = async (pg: SupabaseDirectClient, id: string) => {
```

Comment:
Thanks, this file of helper functions is great. We should do it for every table.